src/App.css

```diff
@@ -0,0 +1,49 @@
+.App {
+  text-align: center;
+  height: 100%;
+
+  /* background: #cf4278;
+  background-image: radial-gradient(rgba(0, 0, 0, 0.205) 1px, transparent 0);
+  background-size: 40px 40px;
+  background-position: -19px -19px; */
+  background-color: #cf4278;
+  background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='48' height='48' viewBox='0 0 48 48'%3E%3Cg fill='%23e04683' fill-opacity='0.4'%3E%3Cpath d='M12 0h18v6h6v6h6v18h-6v6h-6v6H12v-6H6v-6H0V12h6V6h6V0zm12 6h-6v6h-6v6H6v6h6v6h6v6h6v-6h6v-6h6v-6h-6v-6h-6V6zm-6 12h6v6h-6v-6zm24 24h6v6h-6v-6z'%3E%3C/path%3E%3C/g%3E%3C/svg%3E");
+
+  /* background-color:  #eb4d8b; */
```

Comment:
- [ ]  KIndly ensure you delete all commented code in your codebase so that your code can be clean and readable